pkg/interop/json/json.go

```diff
@@ -0,0 +1,14 @@
+/*
+Package json provides various JSON serialization/deserialization routines.
+*/
+package json
+
+// ToJSON serializes value to json. It uses `System.Json.Serialize` syscall.
+func ToJSON(item interface{}) []byte {
+	return nil
+}
+
+// FromJSON deserializes value from json. It uses `System.Json.Deserialize` syscall.
```

Comment:
And it's even more important here, it's going to be used to deserialize random unknown JSONs, I should know how what I'm getting in this `interface{}`.

Reply:
Fixed.